operations/bi_cp.go

```diff
@@ -0,0 +1,41 @@
+package operations
+
+import (
+	"io"
+)
+
+func (bo *BiOperator) Copy(src, dst string) {
+	obj, err := bo.src.Stat(src)
+	if err != nil {
+		bo.errCh <- err
+		return
+	}
+	size := obj.MustGetContentLength()
```

Comment:
Maybe we can check object mode for `src` and `dst` first instead of returning errors during read and write？

Reply:
OK, I'll refactor this logic.